skorch/callbacks/scoring.py

```diff
@@ -64,10 +78,13 @@ def _get_name(self):
             return self.scoring
         if isinstance(self.scoring, partial):
             return self.scoring.func.__name__
+        if isinstance(self.scoring, _BaseScorer):
+            return self.scoring._score_func.__name__
         return self.scoring.__name__
 
     def initialize(self):
         self.best_score_ = np.inf if self.lower_is_better else -np.inf
+        self.scoring = convert_sklearn_metric_function(self.scoring)
```

Comment:
Could you change the new variable to `scoring_`, so that the original `scoring` remains untouched?